locales/en-US/app.ftl

```diff
@@ -1,59 +1,72 @@
 siteName = Firefox Test Pilot
 
+# Page titles, put in the <title> HTML tag.
+[pageTitle]
```

Comment:
Sections use two square brakets. Like this it would generate a parsing error (see [tinker example](http://l20n.github.io/tinker/#eyJzb3VyY2UiOiIjIFBhZ2UgdGl0bGVzLCBwdXQgaW4gdGhlIDx0aXRsZT4gSFRNTCB0YWcuXG5bcGFnZVRpdGxlXVxucGFnZVRpdGxlRGVmYXVsdCA9IEZpcmVmb3ggVGVzdCBQaWxvdFxuIiwiY29udGV4dCI6IiIsImNvbmZpZyI6eyJlc2NhcGVIdG1sIjp0cnVlLCJkaXIiOiJsdHIiLCJsYW5nIjoiZW4tVVMiLCJmaXh0dXJlIjoiZGVmYXVsdCJ9fQ==)).